mergebot_frontend/views.py

```diff
@@ -0,0 +1,89 @@
+from mergebot_frontend import app
+from flask import abort, render_template, send_from_directory
+from mergebot_frontend.models import Poller, QueuedItem, WorkItemStatus
+
+
+@app.route("/")
+def index():
+    """Root handler for website; serves list of pollers.
+    
+    Returns:
+        Index template with the list of active pollers.
+    """
+    pollers = Poller.query.order_by(Poller.project_name).all()
+    return render_template('index.html', pollers=pollers)
+
+
+@app.route('/<proj>')
+def project(proj):
+    """Handler for project page; shows the currently active and pending merges.
+    
+    Args:
+        proj: Name of the project to show.
+
+    Returns:
+        Project template with the currently active and pending merges.
+    """
+    poller = Poller.query.filter_by(
+        project_name=proj).first_or_404()
+    queue = QueuedItem.query.filter_by(
+        project_name=proj,
+    ).order_by(QueuedItem.timestamp.desc()).all()
+    active = None
+    last_start = WorkItemStatus.query.filter_by(
+        project_name=proj,
+        status='START',
+    ).order_by(WorkItemStatus.timestamp.desc()).first()
+    last_finish = WorkItemStatus.query.filter_by(
+        project_name=proj,
+        status='FINISH',
+    ).order_by(WorkItemStatus.timestamp.desc()).first()
+    if last_start and (not last_finish or (last_start.timestamp >
+                                               last_finish.timestamp)):
```

Comment:
Ah yeah, realistically this deserves a comment. What this is saying is "find me the most recent start for this project which doesn't have a finish more recent than it". Breaking it down a bit:
* `if last_start` -> If there are any starts in the DB at all; if not we don't have anything active.
* `not last_finish` -> will be true if we only have memory of one PR and it hasn't finished yet.
* `last_start.timestamp > last_finish.timestamp` -> Most recent start is newer than most recent finish.

So putting it all together, "if any PR has started, and either no PR has finished yet or the most recent start is newer than the most recent finish" set `active` to the most recent start.